### ISSUE TITLE
Simple Fleet Folders

### ISSUE THREAD

**Issue by mondef** [endless-sky/endless-sky#3789]
I've been playing for only a short time, but one thing I've noticed is the very sloppy fleet organization. While I understand the game is not built for huge fleets of a hundred ships or more, I still find it frustrating I can't organize my ships into pre-built fleets. My idea follows:

In the player information screen, we can already reorganize ships by drag and drop. Add a "Create fleet" button to this interface that acts as a folder. You can still drag and drop your ships around to place them into or out of a fleet. Then, add the park and unpack buttons to the fleet folder.

Why could this help? It's a simple organization that would allow players to put their Freighters in a fleet and park them all at once, rather than having to park them one by one. Players could organize a "I'm going to go capture Pirates now" fleet, and a "Time to kill Quarg" feet, and keep their "These freighters are tissue paper" ships out of the way, without spending time parking them all one by one.

**Comment by tehhowch** [comment]
You can already assign ships to a command group, press the associated number key to select them, and then park/unpark as a group.

**Comment by mondef** [comment]
Yes, but this doesn't keep them organized in the player information screen. This is also a hidden feature (meaning you have to know the keyboard shortcuts to use it). I think simple fleet folders wouldn't replace control groups, but and a graphical way to perform a similar function. Would also be easier to look at the player information window (especially if you could collapse the folder).

**Comment by tehhowch** [comment]
Review  https://github.com/endless-sky/endless-sky/issues/2769 and be sure to highlight in your feature request how this is different.
